CONTRIBUTING.md

```diff
@@ -133,14 +133,8 @@ git-cliff --tag 0.18.0 v0.17.2..HEAD | pbcopy
 
 We're [working on automating](https://github.com/temporalio/sdk-typescript/pull/395) the rest of the process:
 
-- Download the artifacts from [GitHub Actions](https://github.com/temporalio/sdk-typescript/actions)
-- Decompress and copy:
-
-```sh
-for f in ~/Downloads/packages-*.zip; do mkdir "$HOME/Downloads/$(basename -s .zip $f)"; (cd "$HOME/Downloads/$(basename -s .zip $f)" && unzip $f && tar -xvzf @temporalio/core-bridge/core-bridge-*.tgz package/releases/ && cp -r package/releases/* ~/gh/release-sdk-typescript/packages/core-bridge/releases/); done
-```
-
 - Log in to npm as `temporal-sdk-team`
+- Download the artifacts from [GitHub Actions](https://github.com/temporalio/sdk-typescript/actions)
 - Publish:
 
 ```sh
```

Comment:
Shall we just break this out into an actual file linked from this readme?

Reply:
I don't actually execute the script, I copy and paste parts at a time 😄 And hopefully I'll be replaced by a robot in https://github.com/temporalio/sdk-typescript/pull/395